tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1454,6 +1454,32 @@ Here are some examples of expressions::
 Identifiers, attributes, and subscripts are valid assignment targets. Other expressions cannot be on the left side of
 an assignment.
 
+self
+""""
+
+``self`` can be used to refer to the current instance and is often equivalent to directly referring to symbols available in
+current script, however, while may seem surprising, it also allows you to access properties, methods and other names in
```

Comment:
Thanks for fixes! Not sure what do you mean by "incomplete sentence"? It continues (and ends) on the line following the last line you quoted.

Reply:
"while may" is an incomplete sentence, you need an object there or it's not a valid sentence :)

Reply:
Meant that the sentence *without* `it` was incomplete, that's what I was correcting :)

Reply:
I guess i'm a "little" worse at English than you are.. I really don't see anything wrong with this sentence, but to avoid this conversation turning into a debate over English grammar i just removed the problematic wording. Hopefully it's correct now 😅

Reply:
It was good with my suggestion :) but no worries it looks better this way